src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/domain/VedtakTilsynBarn.kt

```diff
@@ -15,6 +16,8 @@ sealed interface VedtakTilsynBarn : Vedtaksdata
 
 data class InnvilgelseTilsynBarn(
     val beregningsresultat: BeregningsresultatTilsynBarn,
+    // For vedtak som ble gjort før vi innførte vedtaksperioder lager vi vedtaksperioder fra beregningsresultatet
+    val vedtaksperioder: List<Vedtaksperiode> = beregningsresultat.tilVedtaksperioder(),
```

Comment:
Hjelp meg gjerne litt på om dette kommer til å funke? Har det noen konsekvenser jeg ikke har tenkt på?

Reply:
Ulempen her er at vi ikke får en unik id på en vedtaksperiode, og vet då ikke om den er endret eller ikke
Jeg vil tro at jobben for å patche dataen er verdt det får å unngå at vi trenger å ha kode som mapper fra gammelt format til nytt